editor/src/core/shared/github.ts

```diff
@@ -319,6 +328,15 @@ export const githubFileChangesSelector = createSelector(
   },
 )
 
+export type GithubFileStatus =
+  | 'modified'
+  | 'deleted'
+  | 'untracked'
+  | 'added' // unused, keeping for future reference
```

Comment:
I would err on the side of not including these if they're not being used as it's easy for them to end up living forever in the code.

Reply:
https://github.com/concrete-utopia/utopia/pull/2743/commits/ac58fb13f1b3a147153f72f414a99425d28cfdf1